packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -27,9 +33,25 @@ const BotBuilder = ({
     has_started_onboarding_tour,
     has_started_bot_builder_tour,
     is_preview_on_popup,
+    is_dark_mode_on,
+    selected_strategy_id,
+    loadFileFromRecent,
+    previewRecentStrategy,
 }: TBotBuilder) => {
     const [is_tour_running] = React.useState<boolean>(true);
     const { onMount, onUnmount } = app;
+    const el_ref = React.useRef<HTMLInputElement | null>(null);
+    React.useEffect(() => {
+        setTimeout(() => {
+            previewRecentStrategy(selected_strategy_id);
+        }, 0); // made this async to give it a split second delay
+        if (active_tab === 1) {
+            setColors(is_dark_mode_on);
+            setTimeout(() => {
+                loadFileFromRecent();
+            }, 100);
+        }
```

Comment:
```suggestion
        setTimeout(() => {
            previewRecentStrategy(selected_strategy_id);
        }, 0); // made this async to give it a split second delay
        if (active_tab === 1) {
            setColors(is_dark_mode_on);
            setTimeout(() => {
                loadFileFromRecent();
            }, 100);
        }
```
There are multiple `setTimeout`s here. Is there any way to avoid this?

Reply:
removed setimeout

Reply:
done ✅